clang/lib/AST/ItaniumMangle.cpp

```diff
@@ -2503,7 +2543,20 @@ void CXXNameMangler::mangleType(QualType T) {
   const Type *ty = split.Ty;
 
   bool isSubstitutable =
-    isTypeSubstitutable(quals, ty, Context.getASTContext());
+      isTypeSubstitutable(quals, ty, Context.getASTContext());
```

Comment:
It doesn't seem like you should touch this line.

Reply:
Left as-is